mtgjson5/set_builder.py

```diff
@@ -1008,26 +1009,36 @@ def add_mcm_details(mtgjson_set: MtgjsonSetObject) -> None:
     """
     LOGGER.info(f"Adding MCM details for {mtgjson_set.code}")
     mkm_cards = CardMarketProvider().get_mkm_cards(mtgjson_set.mcm_id)
+
+    if mtgjson_set.mcm_id_extras:
+        extras_cards = CardMarketProvider().get_mkm_cards(mtgjson_set.mcm_id_extras)
+
     for mtgjson_card in mtgjson_set.cards:
         delete_key = False
 
+        if "extendedart" in mtgjson_card.frame_effects and extras_cards:
```

Comment:
i think you should only check for `promo_type=boosterfun` which is an alias for`frame_effects=showcase`, `frame_effects=extendedart`, and `border_color=borderless`

Reply:
Good catch, that solves more cases at once :+1:

Reply:
thanks, however I don't know how the search will behave, for example if there are multiple cards with the same name, except one is showcase and the other one is borderless. Maybe let's wait for @ZeldaZach to chime in

Reply:
Hey folks, thanks for taking the time to bring this to my attention!

Yeah this was my only real concern here. I know upstream has started providing some basic IDs for MCM, which we're now taking advantage of. However, as you can see from my https://github.com/mtgjson/mtgjson/issues/624#issuecomment-699571753, there is a fundamental flaw on the ID system still. I _think_ this PR is unrelated to that issue, and is solely an extension of the service, which is why I'm fine with it as-is.

Reply:
With upstream you mean scryfall?

Yea I dont think this solves all issues with the mcmid but my goal was to fix the extended arts. Thanks to @kodabb it will fix more tho :+1: